packages/verifiable-credentials/src/credentials.ts

```diff
@@ -289,13 +309,16 @@ export default class Credentials {
 		}
 	}
 
-	public async createCredentialJWT({ subjectId, data, schema, context, payload, options }: CreateCredentialJWT): Promise<string> {
+	public async createCredentialJWT(
+		{ subjectId, data, schema, context, payload, options }: CreateCredentialJWT,
```

Comment:
No need to set the default valut of the credentialOptions here, it will eventually get the defaults from `this.createVerifiableCredential`

Reply:
OK, Done